### PR TITLE
Add metrics around equiv item filter logic

### PR DESCRIPTION
**Pull request by Dalamar42**
- This filtering logic currently has a bug (MBST-17731) in that it
  removes items it should not. These metrics are intended to give
  some visibility into how much extra work Cassandra will have to do
  after this is fixed

### CODE REVIEW

File: atlas-cassandra/src/main/java/org/atlasapi/schedule/AbstractEquivalentScheduleStore.java [hunk 1]
```diff
@@ -354,25 +387,28 @@ private String getLockKey(Id channelId, Publisher source, LocalDate date) {
...
             );
 
             if (bestMatch.isPresent()) {
                 Item item = bestMatch.get().copy();
-                item.setBroadcasts(matchingOrEmpty(subjectBroadacast, item.getBroadcasts()));
+                item.setBroadcasts(matchingOrEmpty(subjectBroadcast, item.getBroadcasts()));
                 selected.add(item);
             } else {
                 selected.addAll(matchingOrEmptyBroadcasts(
-                        subjectBroadacast, sourceContent.getValue()
```
Comment (emilssolmanis) [inline atlas-cassandra/src/main/java/org/atlasapi/schedule/AbstractEquivalentScheduleStore.java]: But what if that was supposed to be an Italian accent...? 

luigi

File: atlas-cassandra/src/main/java/org/atlasapi/schedule/AbstractEquivalentScheduleStore.java [hunk 2]
```diff
@@ -49,7 +50,8 @@
 
 public abstract class AbstractEquivalentScheduleStore implements EquivalentScheduleStore {
 
-    private static final Logger log = LoggerFactory.getLogger(AbstractEquivalentScheduleStore.class);
+    private static final Logger log = LoggerFactory.getLogger(AbstractEquivalentScheduleStore
+            .class);
```
Comment (AdamDM) [inline atlas-cassandra/src/main/java/org/atlasapi/schedule/AbstractEquivalentScheduleStore.java]: private static final Logger log = LoggerFactory.getLogger(
                                        AbstractEquivalentScheduleStore.class
                               );

Would this look more normal ^ ?

**Comment by AdamDM** [comment]
olgtm

File: atlas-cassandra/src/main/java/org/atlasapi/schedule/AbstractEquivalentScheduleStore.java (hunk 2 above)
Comment (Dalamar42) [inline atlas-cassandra/src/main/java/org/atlasapi/schedule/AbstractEquivalentScheduleStore.java]: Automatic reformat fail. Fixing

**Pull request merged**